plane-figures-area-calculator.js

```diff
@@ -0,0 +1,33 @@
+const area = (planeFigures) => {
+  if (planeFigures.shape === "circle") {
+    return `${planeFigures.radius ** 2 * Math.PI}`;
+  }
+  if (planeFigures.shape === "square") {
+    return `${planeFigures.side ** 2}`;
+  }
+  if (planeFigures.shape === "rectangle") {
+    return `${planeFigures.horizontalSide * planeFigures.verticalSide}`;
+  }
```

Comment:
You are converting numbers to string unnecessarily. You could just return the numbers directly:

```suggestion
  if (planeFigures.shape === "circle") {
    return planeFigures.radius ** 2 * Math.PI;
  }
  if (planeFigures.shape === "square") {
    return planeFigures.side ** 2;
  }
  if (planeFigures.shape === "rectangle") {
    return planeFigures.horizontalSide * planeFigures.verticalSide;
  }
```